src/utils/calculateDateDifference.ts

```diff
@@ -0,0 +1,13 @@
+import { intervalToDuration } from 'date-fns';
+
+export function calculateDateDifference(startTime: string, endTime: string): string {
```

Comment:
you should type parameters and output example here as well what is this function for